package.json

```diff
@@ -73,6 +73,7 @@
     "@metamask/eslint-config-jest": "^12.1.0",
     "@metamask/eslint-config-nodejs": "^12.1.0",
     "@metamask/eslint-config-typescript": "^12.1.0",
+    "@metamask/snaps-cli": "workspace:^",
```

Comment:
Changes to this file should be reverted

Reply:
Reverted in latest commit.

Reply:
...reverting the changes causes CI to fail now, so i'm not sure what's up.

Reply:
Just needed the yarn.lock to be updated, pushed a fix.